src/test/resources/logback-test.xml

```diff
@@ -1,13 +1,7 @@
 <configuration>
     <include resource="org/springframework/boot/logging/logback/base.xml" />
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
This just tidies our test logback - we were duplicating STDOUT with a slightly different appender pattern for some reason? Anyway, works better without it 👍